server/jira_to_md.go

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+/*
+JIRA Text Formatting Reference
+https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all
+*/
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+type jiraReplacer struct {
+	Type           string
+	RegExp         string
```

Comment:
Maybe to be a little more clear, when I said `make regexp static` I meant to replace `RegExp string` with an (anonymous?) `*regexp.RegExp`and initialize it with a regexp.MustCompile, so all of them will be compiled at load/init time and not every time we receive a webhook. It's not a huge performance penalty to compile these small regex-es every time, but we should "do the right thing" IMO.

Reply:
Totally agree, updated.